CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [x.x.x]
 ### Added
+### Fixed
+### Changed
+
+## [4.30]
+### Added
 - Introduced a `reset dismiss variant` verb
 - Button to reset all dismissed variants for a case
-- Add black border to ideograms
+- Add black border to Chromograph ideograms
```

Comment:
👍

Reply:
> quite agree we shall tick off your list before actually releasing!

It's on clinical-db stage now. I'll do some checks and then continue tomorrow

Reply:
It would be nice with re-uploading an up-to-date MIP case, for the CLI side changes like STR AD..